### PR TITLE
Fix Midnight Train site

### LINKED ISSUE #5: Fix Menu Fonts

### PR DESCRIPTION
**Pull request by qjack001**
Closes #5 and #6, as well as some typo-fixes and adjustments.

### CODE REVIEW

**Review by HoloPollock — APPROVED** [review]

File: _layouts/midnight-train.html [hunk 1]
```diff
@@ -358,6 +358,11 @@
 			font-weight: normal;
 			text-transform: uppercase
 		}
+
+		h1 svg
+		{
+			max-width: 500px;;
```
Comment (HoloPollock) [inline _layouts/midnight-train.html:364]: This isn't like an "issue but can you remove the ;

**Review by HoloPollock — CHANGES_REQUESTED** [review]

File: _layouts/midnight-train.html [hunk 2]
```diff
@@ -441,11 +446,15 @@
...
 			text-transform: uppercase;
 		}
-		 h3 {
+
+		h3
+		{
 			width: 100%;
 			max-width: 50rem;
-			margin: 0.5rem auto 0.5rem;
-			font-family: "heading";
```
Comment (HoloPollock) [inline _layouts/midnight-train.html:448]: Why did you remove this?

Comment (qjack001) [reply _layouts/midnight-train.html:448]: I found the heading font wasn't good looking at such small sizes, so I swapped it out for the body font bolded and capatilized.

File: _layouts/midnight-train.html (hunk 1 above)
Comment (qjack001) [reply _layouts/midnight-train.html:364]: ya whoops

**Review by HoloPollock — APPROVED** [review]
